app/assets/javascripts/compute_resources/ovirt/nic_info.js

```diff
@@ -1,3 +1,3 @@
 providerSpecificNICInfo = function(form) {
-  return form.find('.ovirt_name').val() + ' @ ' + form.find('.ovirt_network').val();
+  return form.find('.ovirt_name').val() + ' @ ' + form.find('.ovirt_network').text();
```

Comment:
It fixes my own mistake. Value in the selectbox is some hash, while the text is name of the network. So without this change the js used to produce something like `eth0 @ b3499d8ae083` now it's `eth0 @ local_network`